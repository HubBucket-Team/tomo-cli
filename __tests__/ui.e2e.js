import execa from 'execa';
import {
    useTemporaryDirectory
} from './tomo-test';

const MAX_TIMEOUT = 300000;
jest.setTimeout(MAX_TIMEOUT);

describe('tomo', () => {
    let tempDirectory;
    const [setTempDir, cleanupTempDir] = useTemporaryDirectory();
    beforeEach(async () => {
        tempDirectory = await setTempDir();
        process.chdir(tempDirectory);
    });
    afterEach(async () => {
        await cleanupTempDir();
    });
    test('(no command or terms)', async () => {
        expect(process.cwd()).toBe(tempDirectory);
        const {stdout} = await execa('tomo');
        expect(stdout).toMatchSnapshot();
    });
    xtest('add', async () => {
        expect(process.cwd()).toBe(tempDirectory);
        // const {stdout} = await execa('tomo', ['add']);
        // expect(stdout).toMatchSnapshot();
    });
    test('add eslint --skip-install', async () => {
        expect(process.cwd()).toBe(tempDirectory);
        // const {stdout} = await execa('tomo', ['add', 'eslint', '--skip-install']);
        // console.log(stdout);
    });
});