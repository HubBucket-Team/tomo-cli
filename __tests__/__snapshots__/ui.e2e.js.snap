// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tomo (no command or terms) 1`] = `
"
 ┌────────────────────┐
 │ [33m[1mUNDER CONSTRUCTION[22m[39m │
 └────────────────────┘
"
`;
